src/ScriptSymlinker.php

```diff
@@ -56,7 +56,9 @@ public static function createSymlinks(Event $event)
 
                     $event->getIO()->write("  Symlinking <comment>$targetPath</comment> to <comment>$linkPath</comment>");
                     $fs->ensureDirectoryExists(dirname($linkPath));
-                    $fs->relativeSymlink($targetPath, $linkPath);
+                    if (!$fs->relativeSymlink($targetPath, $linkPath) && !is_dir($targetPath)) {
```

Comment:
What is the reason for cheking that `$targetPath` is not a directory?

Reply:
Because PHP's copy-function cannot copy directories.

Reply:
Ok, but why don't you use the `Composer\Util\Filesystem::copy` function as you proposed in #2

Reply:
Because you're using composer 1.0 and `Composer\Util\Filesystem::copy` is added recently in a newer version. For me, the impact of upgrading to an newer version of composer is not clear and there are no unit tests the verify the library is still working as expected after an upgrade.

Reply:
I see, but I'd prefer not to offer partial support copying only files.
We can wait for the next composer release (upgrading the composer requirement is not a problem here), or provide a custom copy method with support for directories, that can be removed in the future. What do you think?